### ISSUE TITLE
Moonraker: sudo password required

### ISSUE THREAD

**Issue by leigh-johnson** [bitsy-ai/printnanny-os#144]
## **Describe the bug**
Moonraker is showing the following warnings:

```
Sudo Password Required
Moonraker requires sudo access to finish updating. Please click on the attached link and follow the instructions.
```
```
Moonraker warning
Moonraker requires sudo permission to update the system service. Please check your notifications for further instructions.
```

I have a feeling moonraker is running some random command to test permissions, so let's figure out what that is and add to: 
https://github.com/bitsy-ai/meta-bitsy/blob/main/meta-printnanny/recipes-extended/sudo/files/000_alias

Polkit rules should provide all other permissions: https://github.com/bitsy-ai/meta-bitsy/blob/main/meta-printnanny/recipes-3dprinter/moonraker/files/moonraker.rules

## Issue.txt

```
-----------------------
Build Info:  |
-----------------------
IMAGE_NAME = printnanny-release-image-raspberrypi4-64-20221114160405
DISTRO_PRETTY = PrintNanny Linux 0.4.3 (Dunite)
VARIANT_NAME = PrintNanny OS Core
VARIANT_ID = core
DISTRO = printnanny
DISTRO_NAME = PrintNanny Linux
DISTRO_VERSION = 0.4.3
DISTRO_CODENAME = Dunite
ID_LIKE = BitsyLinux
HOME_URL = https://printnanny.ai
BUG_REPORT_URL = https://github.com/bitsy-ai/printnanny-os/issues
YOCTO_VERSION = 4.0.2
YOCTO_CODENAME = Langdale
MACHINE = raspberrypi4-64
TUNE_PKGARCH = cortexa72
-----------------------
Layer Revisions:      |
-----------------------
meta              = langdale:6b9db5a99bf0fab47219d4648f4663c4397ac87f
meta-poky         = langdale:6b9db5a99bf0fab47219d4648f4663c4397ac87f
meta-yocto-bsp    = langdale:6b9db5a99bf0fab47219d4648f4663c4397ac87f
meta-raspberrypi  = master:722c51647c75d63e8c6c53ee214c4e3f0a263ed2
meta-oe           = langdale:c5668905a6d8a78fb72c2cbf8b20e91e686ceb86
meta-python       = langdale:c5668905a6d8a78fb72c2cbf8b20e91e686ceb86
meta-multimedia   = langdale:c5668905a6d8a78fb72c2cbf8b20e91e686ceb86
meta-bitsy        = issue-143-polkit:c5a6d153980fefa9237cd2b207e762b6633aba4b
meta-neural-network = langdale-compat:c8422a7380ef81f75bb5ca2216210b434c4e7114
meta-printnanny   = issue-143-polkit:c5a6d153980fefa9237cd2b207e762b6633aba4b
meta-networking   = langdale:c5668905a6d8a78fb72c2cbf8b20e91e686ceb86
meta-filesystems  = langdale:c5668905a6d8a78fb72c2cbf8b20e91e686ceb86
meta-initramfs    = langdale:c5668905a6d8a78fb72c2cbf8b20e91e686ceb86
meta-webserver    = langdale:c5668905a6d8a78fb72c2cbf8b20e91e686ceb86
meta-swupdate     = master:5e611f44db1b9333342ed61d77a3c3ed7be69c2b
workspace         = langdale:6b9db5a99bf0fab47219d4648f4663c4397ac87f
meta-microcontroller = master:9f396e595b2d639be7f2b494f4a61e48d9495058
```

**Comment by leigh-johnson** [comment]
```
Nov 14 08:29:48 pn-v0-4 bash[817]: [moonraker.py:_initialize_component()] - Performing Component Post Init: [authorization]
Nov 14 08:29:48 pn-v0-4 sudo[843]: printnanny : PWD=/var/lib/moonraker ; USER=root ; COMMAND=/bin/systemctl --version
Nov 14 08:29:48 pn-v0-4 bash[817]: [machine.py:_check_service_file()] - Performing Service Validation...
Nov 14 08:29:49 pn-v0-4 bash[817]: [moonraker.py:add_warning()] - Moonraker requires sudo permission to update the system service. Please check your notifications for further intructions.
```

**Comment by leigh-johnson** [comment]
:tada: Done! Fix will be available in v0.4.4
